server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
 
 Existing user::
 
-When existing users try to log in, they are required to register their WebAuthn authenticator automatically :
+In case that `WebAuthn Authenticator` is setup as required as shown in the first example, then when existing users try to log in,
```

Comment:
If `WebAuthn Authenticator` is set up as required as shown in the first example, then when existing users try to log in,